pkg/network/driver/nmstate/types.go

```diff
@@ -97,6 +98,13 @@ type TapDevice struct {
 	GID    int `json:"GID,omitempty"`
 }
 
+type MacvtapDevice struct {
```

Comment:
It should be added to the integration tests (even though it cannot be executed at the moment because we are missing the virt-chroot binary in the running container).